spectrum_utils/plot.py

```diff
@@ -227,6 +232,111 @@ def spectrum(
     return ax
 
 
+def mass_errors(
+    spec: MsmsSpectrum,
+    *,
+    plot_unknown: bool = True,
+    color_ions: bool = True,
+    grid: Union[bool, str] = True,
+    ax: Optional[plt.Axes] = None,
+) -> plt.Axes:
+    """
+    Plot mass error bubble plot for a given spectrum.
+
+    Parameters
+    ----------
+    spec : MsmsSpectrum
+        The spectrum with mass errors to be plotted.
+    plot_unknown : bool, optional
+        Flag indicating whether or not to plot mass errors for unknown peaks.
+    color_ions : bool, optional
+        Flag indicating whether or not to color dots for annotated fragment
+        ions. The default is True.
+    grid : Union[bool, str], optional
+        Draw grid lines or not. Either a boolean to enable/disable both major
+        and minor grid lines or 'major'/'minor' to enable major or minor grid
+        lines respectively.
+    ax : Optional[plt.Axes], optional
+        Axes instance on which to plot the mass errors. If None the current
+        Axes instance is used.
+
+    Returns
+    -------
+    plt.Axes
+        The matplotlib Axes instance on which the mass errors are plotted.
+
+    Notes
+    -----
+    The mass error bubble plot was first introduced in [1]_.
+
+    References
+    ----------
+    .. [1] Barsnes,H., Eidhammer,I. and Martens,L. (2010)
+       FragmentationAnalyzer: An open-source tool to analyze MS/MS
+       fragmentation data. PROTEOMICS, 10, 1087–1090.
+       doi:10.1002/pmic.200900681
+
+    """
+    if ax is None:
+        ax = plt.gca()
+
+    _format_ax(ax, grid)
+    ax.set_ylabel("mass error")
+
+    if len(spec.mz) == 0:
+        return ax
+
+    round_mz = 50
```

Comment:
**todo:** Maybe this can be extracted to a very simple `_get_xlim(MsmsSpectrum) -> Tuple[float, float]` function so that the functionality doesn't need to be duplicated here and in `spectrum(...)`.